adviser/base/adviser-sa.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: adviser
```

Comment:
which component would use this sa ?

Reply:
this will be in the adviser workflow. I was looking into it though and I think I can patch the pod definition for a single workflow step so that we don't have to run the whole adviser workflow with elevated priviledges